opentracing-shim/src/main/java/io/opentelemetry/opentracingshim/Propagators.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.opentracingshim;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.context.propagation.TextMapPropagator;
+
+/**
+ * Container for {@link io.opentracing.propagation.Format.Builtin#TEXT_MAP} and {@link
+ * io.opentracing.propagation.Format.Builtin#HTTP_HEADERS} format propagators.
+ */
+public class Propagators {
+  private final TextMapPropagator textMapPropagator;
+  private final TextMapPropagator httpHeadersPropagator;
+
+  public TextMapPropagator textMapPropagator() {
+    return textMapPropagator;
+  }
+
+  public TextMapPropagator httpHeadersPropagator() {
+    return httpHeadersPropagator;
+  }
+
+  public Propagators(TextMapPropagator textMapPropagator, TextMapPropagator httpHeadersPropagator) {
+    this.textMapPropagator = textMapPropagator;
+    this.httpHeadersPropagator = httpHeadersPropagator;
+  }
+
+  public static PropagatorsBuilder builder() {
+    return new PropagatorsBuilder();
+  }
+
+  public static class PropagatorsBuilder {
```

Comment:
Move it out to be top level, instead of `Propagators.PropagatorsBuilder`? These days in OTel Java the builders are not exposed as (nested) static classes, so worth stay uniform there.

Reply:
(But keep the creation through `Propagators.builder()` ;) )

Reply:
done